EDAM_dev.owl

```diff
@@ -37269,6 +37269,33 @@ experiments employing a combination of technologies.</oboInOwl:hasDefinition>
     
 
 
+    <!-- http://edamontology.org/format_XXXX -->
+
+    <owl:Class rdf:about="http://edamontology.org/format_XXXX">
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_2921"/>
+        <rdfs:subClassOf rdf:resource="http://edamontology.org/format_3016"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://edamontology.org/is_format_of"/>
+                <owl:someValuesFrom rdf:resource="http://edamontology.org/data_3498"/>
```

Comment:
Special appreciation for adding the _is_format_of_! 🚀🙌🏽😸

Reply:
:grin: I made sure to read the [documentation](https://edamontologydocs.readthedocs.io/en/latest/developers_guide.html#mandatory-attributes) about attributes, especially formats